modules/cnv-cancelling-vm-migration-web.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-cancel-vmi-migration.adoc
+
+[id="cnv-cancelling-vm-migration-web_{context}"]
+= Cancelling live migration of a virtual machine instance in the web console
+
+A live migration of the virtual machine instance can be cancelled using the 
+&#8942; button found on each virtual machine in the *Workloads* -> *Virtual Machines* 
```

Comment:
kebap button (&#8942;) 

might be more descriptive

Reply:
The 'kebab/kebap' menu conversation came up in OCP docs and I'm waiting to hear what the decision was. 
I'd be more inclined to call it an 'Actions' menu, since it mostly mirrors the options available for that button on the <Object> Description page

Reply:
+1 Action menu sounds good